plugins/explore/src/components/GroupsExplorerContent/GroupsExplorerContent.tsx

```diff
@@ -29,8 +38,10 @@ type GroupsExplorerContentProps = {
 export const GroupsExplorerContent = ({
   title,
 }: GroupsExplorerContentProps) => {
+  const classes = useStyles();
+
   return (
-    <Content noPadding>
+    <Content noPadding stretch className={classes.root}>
```

Comment:
what does it mean to stretch AND have those styles?

Reply:
Stretch only configures flex box for the elements inside `<Content/>`, the styles set the size of the content area itself. The max height is required as the SVG element likes to grow bigger then height. The whole layout thing isn't that nice to solve here… And I might missed some edge cases.